src/utils/image.cpp

```diff
@@ -238,4 +238,49 @@ void save_image(const std::string& filename, const CPUMat& src,
   save_image(filename, cv_mat, dims);
 }
 
+El::Matrix<uint8_t> get_uint8_t_image(const lbann::CPUMat& image,
```

Comment:
This extracted function's lines in the previous function should be deleted and replaced with a call to this function.